portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -12,8 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-async function fetchMessage() {
-    const response = await fetch('/data');
-    const message = await response.json();
-    document.getElementById("fetchmessage-container").innerText = message;
-}
+package com.google.sps.data;
+
+// Represents a single comment with a username and the content of the comment.
+public class Comment {
+    private String username;
```

Comment:
If you don't plan to modify the value of username and comment, then you can declare them to be final values (i.e. immutable)

private final String username;
private final String comment;

Reply:
Makes sense, thanks.